ui/src/index.ts

```diff
@@ -198,20 +200,23 @@ window.addEventListener("load", async function() {
       }
       changePageTo(thankYouPage);
     };
-    submitGreat.onclick = () => saveResponse("great");
-    submitNotThatGreat.onclick = () => saveResponse("notThatGreat");
-    submitNotGreatAtAll.onclick = () => saveResponse("notGreatAtAll");
+
+    const comment = (<HTMLTextAreaElement>commentTextarea).value;
```

Comment:
I would be better to cast at initialization time (line 30), not at use time (here). Also this cast syntax has been deprecated in favor of `value as type`.